src/views/checkups/network/list/CheckupsNetworkListEmptyState.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import ExternalLink from '@kubevirt-utils/components/ExternalLink/ExternalLink';
+import { ALL_NAMESPACES_SESSION_KEY } from '@kubevirt-utils/hooks/constants';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Button,
+  ButtonVariant,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStatePrimary,
+  EmptyStateSecondaryActions,
+  EmptyStateVariant,
+  Title,
+} from '@patternfly/react-core';
+import { NetworkIcon } from '@patternfly/react-icons';
+import { createURL } from '@virtualmachines/details/tabs/overview/utils/utils';
+
+import { installOrRemoveCheckupsNetworkPermissions } from '../utils/utils';
+
+import './checkups-network-list-empty-state.scss';
+
+const CheckupsNetworkListEmptyState = ({ isPermitted, nadsInNamespace }) => {
+  const { t } = useKubevirtTranslation();
+  const history = useHistory();
+  const [namespace] = useActiveNamespace();
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  return (
+    <EmptyState variant={EmptyStateVariant.large}>
+      <EmptyStateIcon icon={NetworkIcon} />
+      <Title headingLevel="h4" size="lg">
+        {t('No network latency checkups yet')}
+      </Title>
+      <EmptyStateBody>
+        {t('To get started, install permission and then run a checkup')}
```

Comment:
You're missing the 's' on 'permissions'.

Reply:
will add